src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +56,10 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    
+	@Override
+	public String getPrintableString(Printable... printables) {
+		// TODO Auto-generated method stub
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.